freeipa/src/main/java/com/sequenceiq/freeipa/service/upgrade/ccm/CcmParametersConfigService.java

```diff
@@ -0,0 +1,47 @@
+package com.sequenceiq.freeipa.service.upgrade.ccm;
+
+import static java.util.Collections.singletonMap;
+import static org.apache.commons.lang3.StringUtils.EMPTY;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import org.springframework.stereotype.Service;
+
+import com.sequenceiq.cloudbreak.auth.altus.EntitlementService;
+import com.sequenceiq.cloudbreak.ccm.cloudinit.CcmV2JumpgateParameters;
+import com.sequenceiq.cloudbreak.orchestrator.model.SaltPillarProperties;
+
+@Service
+public class CcmParametersConfigService {
+
+    private static final String UPGRADE_CCM_KEY = "ccm_jumpgate";
+
+    private static final String UPGRADE_CCM_SLS_PATH = "/upgradeccm/init.sls";
+
+    @Inject
+    private EntitlementService entitlementService;
+
+    public Map<String, SaltPillarProperties> createCcmParametersPillarConfig(String accountId, CcmV2JumpgateParameters ccmParameters) {
+        if (ccmParameters != null) {
```

Comment:
could you add logs to the if and else branch

Reply:
sure